os_migrate/plugins/module_utils/const.py

```diff
@@ -11,7 +11,7 @@
 RES_TYPE_IMAGE = 'openstack.image.Image'
 RES_TYPE_KEYPAIR = 'openstack.compute.Keypair'
 RES_TYPE_NETWORK = 'openstack.network.Network'
-RES_TYPE_USER_PROJECT_ROLE_ASSIGNMENT = 'openstack.identity.UserProjectRoleAssignment'
+RES_TYPE_USER_PROJECT_ROLE_ASSIGNMENT = 'openstack.identity.RoleAssignment'
```

Comment:
I think this should stay "UserProjectRoleAssignment" as it was before, we're specifying the resource type as seen by OS Migrate. Usually it maps to OpenStack resources 1:1 but it does not have to, e.g. the OS Migrate "RouterInterface" type maps to OpenStack's "Port" type. Every router interface is a port, but not every port is a router interface.

Similarly here - every UserProjectRoleAssignment is an OpenStack role assignment, but not every OpenStack role assignment is between a user-project-role triplet, which is what we're handling here.

This rename can be done in a separate mini-patch. The rest of the PR looks good, thanks for implementing this!